src/Ogooreck.Sample.BusinessLogic.FSharp.Tests/Deciders/BankAccountDecider.fs

```diff
@@ -0,0 +1,80 @@
+﻿module Deciders.BankAccountDecider
+
+open System
+open BankAccount
+open BankAccountPrimitives
+
+type OpenBankAccount =
+    { BankAccountId: AccountId
+      AccountNumber: AccountNumber
+      ClientId: ClientId
+      CurrencyISOCode: CurrencyCode
```

Comment:
```suggestion
      CurrencyIsoCode: CurrencyCode
```

Reply:
Updated in https://github.com/oskardudycz/Ogooreck/pull/13/commits/9c3bd4fa7efd66b75bf111513ba31eea4892e0cb.